src/build/checkstyle.xml

```diff
@@ -66,6 +66,27 @@
 		<module name="ArrayTypeStyle"/>
 		<module name="TodoComment"/>
 		<module name="UpperEll"/>
+		<module name="NeedBraces"/>
+		<module name="AvoidStarImport"/>
+		<module name="ConstantName"/>
+		<!--<module name="DeclarationOrder"/>-->
+		<module name="EqualsAvoidNull"/>
+		<!--<module name="HiddenField"/>-->
+		<module name="InnerTypeLast"/>
+		<module name="MissingDeprecated"/>
+		<module name="MissingOverride"/>
+		<module name="MutableException"/>
+		<module name="NoClone"/>
+		<module name="NoFinalizer"/>
+		<module name="OneStatementPerLine"/>
+		<module name="OneTopLevelClass"/>
+		<module name="OuterTypeFilename"/>
+		<module name="ParameterAssignment"/>
+		<module name="StringLiteralEquality"/>
+		<module name="SuperClone"/>
+		<module name="SuperFinalize"/>
+		<!--<module name="TrailingComment"/>-->
+		<module name="VariableDeclarationUsageDistance"/>
```

Comment:
What's the basis for this ruleset? Is this just Sun/Oracle Java conventions?

Reply:
Checkstyle offerings and what I experience as the concensus among java devs at the clients I've worked for... Do you mean some specific rules or the whole lot aside from the few mentioned explicitly so far?

Reply:
I was looking at the whole.

Ok so this is drawn from experience / personal taste. That's fine, I think I agree with the changes you've made.